zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -347,20 +348,25 @@ def _get_temp_dst(
         )
         return attrs.evolve(dst.with_procs(read_procs=()), backend=backend_temp)
 
+    def _make_task(
+        self,
+        arg: Tuple[
+            VolumetricIndex, VolumetricBasedLayerProtocol, dict[str, Any]
+        ],  # cannot type with P.kwargs
+    ) -> mazepa.tasks.Task[R_co]:
+        return self.op.make_task(idx=arg[0], dst=arg[1], **arg[2])
+
     def make_tasks_without_checkerboarding(
         self,
         idx_chunks: Iterable[VolumetricIndex],
         dst: VolumetricBasedLayerProtocol,
         op_kwargs: P.kwargs,
     ) -> List[mazepa.tasks.Task[R_co]]:
-        tasks = [
-            self.op.make_task(
-                idx=idx_chunk,
-                dst=dst,
-                **op_kwargs,
+        with multiprocessing.Pool() as pool_obj:
+            tasks = pool_obj.map(
+                self._make_task,
```

Comment:
Why is the `self._make_task` necessary here? Can we just map `self.op.make_task`?

Reply:
Cannot because parallel map requires a single argument.

Reply:
Gotcha. In that case it's fine for the helper to be not typed tightly